src/test/java/seedu/duke/parser/ParserTest.java

```diff
@@ -13,11 +15,30 @@
 class ParserTest {
 
     @Test
-    void parseTemplate_LrTemplate_returnsTemplateCommand() throws DukeException {
+    void parse_noLine_expectException() {
         Parser p = new Parser();
-        String line = "template rl";
+        String line = "";
+        assertThrows(DukeException.class, () -> p.parse(line));
+    }
+
+    @Test
+    void parseHelp_help_returnsHelpCommand() throws DukeException {
+        Parser p = new Parser();
+        String line = "help";
         Command c = p.parse(line);
-        assertTrue(c instanceof TemplateCommand);
+        assertTrue(c instanceof Command); // change to HelpCommand once done
+    }
+
+    @Test
+    void parseTemplate_Template_returnsTemplateCommand() throws DukeException {
+        Parser p = new Parser();
+        String line = "template ";
+        String[] tmp = {"r", "rl", "rc", "lc"};
+        Command c;
+        for (String s : tmp) {
```

Comment:
If you commit the previous suggestion you'll have to change this too:
```suggestion
        for (String s : templates) {
```